test/remote/gateways/remote_element_test.rb

```diff
@@ -41,6 +42,12 @@ def test_successful_purchase_with_payment_account_token
     assert_equal 'Approved', response.message
   end
 
+  def test_successful_purchase_with_shipping_address
+    response = @gateway.purchase(@amount, @credit_card, @options.merge(shipping_address: address(address1: "Shipping")))
+    assert_success response
+    assert_equal 'Approved', response.message
+  end
```

Comment:
@davidsantoso In looking into that, I realized the shipping fields needed to go in the same element as the billing address fields. But apparently even after that fix, no shipping fields are returned in a response. Now I'm not sure if that's intended or if it's still not being passed correctly. Investigating...